tools/executiontester/assetcontract/assetcontract.go

```diff
@@ -24,6 +24,7 @@ type ReturnedContract struct {
 	Erc721Token blockchain.Erc721Token
 	ArrDelete   blockchain.ArrayDelete
 	ArrString   blockchain.ArrayString
+	ArrPassing  blockchain.ArrayPassing
```

Comment:
can you combine this passing test, code, solidity binary into existing ArrDelete?

the function is much similar, pushing int value into an array, and delete one item in the array

Reply:
ok